packages/core/src/common/styles.ts

```diff
@@ -159,7 +160,11 @@ export function mergeAndRealizeTheme(theme: Theme, ...overlays: Partial<Theme |
             for (const key in overlay) {
                 // eslint-disable-next-line no-prototype-builtins
                 if (overlay.hasOwnProperty(key)) {
-                    merged[key] = (overlay as any)[key];
+                    if (key === "bgCell") {
+                        merged[key] = blend(overlay[key] as string, merged[key]);
```

Comment:
Without doing this we retain the previous requirement that bgCell always be a solid color. Now only the main theme need be a solid color.